.coafile

```diff
@@ -1,7 +1,7 @@
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = **/*.rst, README.md
+files = **/*.rst
```

Comment:
you sure? `./README.rst` contains a slash. Paths allow redundant slashes as well so even `.//README.rst` should match it - if it doesn't that would be a globbing bug.

CC @fneu about this behaviour